aws-cloudformation-stackset/resource-role.yaml

```
@@ -0,0 +1,44 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This CloudFormation template creates a role assumed by CloudFormation
+  during CRUDL operations to mutate resources on behalf of the customer.
+
+Resources:
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 8400
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: resources.cloudformation.amazonaws.com
+            Action: sts:AssumeRole
+      Path: "/"
+      Policies:
+        - PolicyName: ResourceTypePolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                - "cloudformation:CreateStackInstances"
+                - "cloudformation:CreateStackSet"
+                - "cloudformation:DeleteStackInstances"
+                - "cloudformation:DeleteStackSet"
+                - "cloudformation:DescribeStackInstance"
+                - "cloudformation:DescribeStackSet"
+                - "cloudformation:DescribeStackSetOperation"
+                - "cloudformation:ListStackInstances"
+                - "cloudformation:ListStackSets"
+                - "cloudformation:TagResource"
+                - "cloudformation:UntagResource"
+                - "cloudformation:UpdateStackSet"
+                - "cloudformation:UpdatetackInstances"
```

```suggestion
                - "cloudformation:UpdateStackInstances"
```
https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awscloudformation.html#awscloudformation-actions-as-permissions

---

Nice catch. Will fix it.